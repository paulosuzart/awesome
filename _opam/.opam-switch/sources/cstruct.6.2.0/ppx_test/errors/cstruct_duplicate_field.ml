[%%cstruct
type dup = {
  x : uint8_t;
  y : uint8_t;
  x : uint8_t;
} [@@little_endian]
]
