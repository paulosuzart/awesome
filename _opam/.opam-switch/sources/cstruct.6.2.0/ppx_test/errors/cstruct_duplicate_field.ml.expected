Error: ppx_cstruct: field x is present several times in this type
